.github/workflows/test.yaml

```diff
@@ -74,4 +74,4 @@ jobs:
       - run: cargo build --bin lalrpop --features pico-args
       - name: Run feature powerset check
         # test with minimal amount of features plus a few extra on regex/regex-syntax
-        run: cargo hack test --workspace --feature-powerset --exclude-features pico-args,default --optional-deps
+        run: cargo hack test --workspace --feature-powerset --exclude-features pico-args,default --optional-deps --mutually-exclusive-features test-set,test-not-set
```

Comment:
Cool flag that I didn't know about.  I think this means we can revert most of 8c65638baba6517897d31ecb7af3d8a6a165833b?

All the `cfg` changes in there are redundant now, but as far as I can tell `use_cfg_created_parser()` wasn't being called before I added a test case for it.

Reply:
Yeah, it's a little hidden on the README page.

I guess it depends whether we want to explicitly test for the error as opposed to just avoiding it(which is what I thought you were going for)?

Reply:
But with this change we don't explicitly test for the error anymore, right?  This causes feature-powerset to just not use these two features together, so the error check never comes up.

Reply:
Sure, lets keep that behavior.